ledger_device_ui_sdk/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "ledger_device_ui_sdk"
```

Comment:
No rust in the name of this crate ?

Reply:
technically as this will only be used in rs file and cargo file this is not really necessary? We could even remove it from the sdk crate (but keep it in the repository name?)

Reply:
no 😞 but it may be integrated into `ledger_device_rust_sdk` in the future. And, in the meantime, `ledger_device_ui_rust_sdk` is too verbose IMHO....

Reply:
why not, in that case, we would have:

- ledger_device_sdk
- ledger_device_ui_sdk
- ledger_secure_sdk_sys
- include_gif

Is it ok for everyone ?

Reply:
This is the moment to do the renaming if we want to guys... ^^

Reply:
I'm ok with that